src/Sulu/Bundle/AdminBundle/Resources/js/containers/FieldBlocks/FieldBlocks.js

```diff
@@ -57,6 +76,8 @@ export default class FieldBlocks extends React.Component<FieldTypeProps<Array<Bl
         const newValues = toJS(oldValues);
         jsonpointer.set(newValues[index], '/' + name, value);
 
+        this.updateValue(newValues);
```

Comment:
Just for my understanding: This is absolutely necessary, right? The `componentDidUpdate` call you've also touched is not executed between multiple `onChange` calls, right?

Reply:
Yep, we debugged through it and the `componentDidUpdate` is called only after the second `onChange` is called. Thus the `value` is not yet updated during the second `onChange` call.